examples/charts/area.py

```diff
@@ -1,6 +1,7 @@
 from collections import OrderedDict
-from bokeh.charts import Area
-from bokeh.plotting import show, output_file
+
+from bokeh.charts import Area, show, output_file
```

Comment:
well.. it we are planning on having `show` and friends only on `state` in a future release then I agree with you. We should promote it from now..